src/map/script.cpp

```diff
@@ -28027,7 +28000,6 @@ struct script_function buildin_func[] = {
 	BUILDIN_DEF(playBGMall,"s?????"),
 	BUILDIN_DEF(soundeffect,"si"),
 	BUILDIN_DEF(soundeffectall,"si?????"),	// SoundEffectAll [Codemaster]
-	BUILDIN_DEF(strmobinfo,"ii"),	// display mob data [Valaris]
```

Comment:
Please restore it by using BUILDIN_DEF_DEPRECATED
I think the general rule is, that we add the deprecation date here and then delete it after 2 years.
@Atemo please correct me.

Reply:
We didn't really define a removal date until now, but I agree with you on at least 2 years.

`BUILDIN_DEF_DEPRECATED(strmobinfo,"ii", "2025-03-11"),`